llm_demo/orchestrator/langchain_tools/tools.py

```diff
@@ -155,13 +155,24 @@ async def list_flights(
 
 class QueryInput(BaseModel):
     query: str = Field(description="Search query")
+    filter_time: str = Field(description="Time for filtering amenities operating hours")
```

Comment:
nit: Consider renaming "open_time" and "open_day". It's the time/day we want them to be open. 
```suggestion
    filter_time: str = Field(description="Time for filtering amenities by operating hours.")
```